shared/style/status.css

```diff
@@ -6,9 +6,7 @@ html[dir="ltr"] section[role="status"] [dir] { text-align: left; }
 html[dir="rtl"] section[role="status"] [dir] { text-align: right; }
 
 section[role="status"] {
-  background-color: rgba(64,64,64,1);
-  background-image: url(status/images/ui/pattern.png);
```

Comment:
Seems like this is no longer used. Please remove the image files if that's the case.